EngineLayer/Mods/fallOff.txt

```diff
@@ -1,253 +1,260 @@
-fallOffN
-ID   fallOffN W
-PP   Peptide N-terminal.
-TG   W
-MT   fallOffN
-CF   C-11H-10N-2O-1
-//
-ID   fallOffN Y
-PP   Peptide N-terminal.
-TG   Y
-MT   fallOffN
-CF   C-9H-9N-1O-2
-//
-ID   fallOffN R
-PP   Peptide N-terminal.
-TG   R
-MT   fallOffN
-CF   C-6H-12N-4O-1
-//
-ID   fallOffN U
-PP   Peptide N-terminal.
-TG   U
-MT   fallOffN
-CF   C-3H-5N-1O-1Se-1
-//
-ID   fallOffN F
-PP   Peptide N-terminal.
-TG   F
-MT   fallOffN
-CF   C-9H-9N-1O-1
-//
-ID   fallOffN H
-PP   Peptide N-terminal.
-TG   H
-MT   fallOffN
-CF   C-6H-7N-3O-1
-//
-ID   fallOffN M
-PP   Peptide N-terminal.
-TG   M
-MT   fallOffN
-CF   C-5H-9N-1O-1S-1
-//
-ID   fallOffN E
-PP   Peptide N-terminal.
-TG   E
-MT   fallOffN
-CF   C-5H-7N-1O-3
-//
-ID   fallOffN K
-PP   Peptide N-terminal.
-TG   K
-MT   fallOffN
-CF   C-6H-12N-2O-1
-//
-ID   fallOffN Q
-PP   Peptide N-terminal.
-TG   Q
-MT   fallOffN
-CF   C-5H-8N-2O-2
-//
-ID   fallOffN D
-PP   Peptide N-terminal.
-TG   D
-MT   fallOffN
-CF   C-4H-5N-1O-3
-//
-ID   fallOffN N
-PP   Peptide N-terminal.
-TG   N
-MT   fallOffN
-CF   C-4H-6N-2O-2
-//
-ID   fallOffN I
-PP   Peptide N-terminal.
-TG   I
-MT   fallOffN
-CF   C-6H-11N-1O-1
-//
-ID   fallOffN L
-PP   Peptide N-terminal.
-TG   L
-MT   fallOffN
-CF   C-6H-11N-1O-1
-//
-ID   fallOffN C
-PP   Peptide N-terminal.
+Glycosylations
```

Comment:
I'd like for us to discuss what we do with glyco mods and how we deal with their neutral losses